src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,159 @@
+package duke;
+
+import duke.command.*;
+
+import java.util.Arrays;
+
+/**
+ * Parser parses and helps to make sense of user input.
+ */
+public class Parser {
+    private final static String INDENTATION = "   ";
+
+    private enum Commands {
+        BYE, LIST, MARK, UNMARK, TODO, DEADLINE, EVENT, DELETE, FIND
+    }
```

Comment:
Nice use of enums here, perhaps you could consider abstracting them out to a separate file? This might be useful if you ever needed them elsewhere in your codebase.